### ISSUE TITLE
Error calling macro, no module found

### ISSUE THREAD

**Issue by SirStepheno** [xlwings/xlwings#2144]
#### OS (e.g. Windows 10 or macOS Sierra)'
Windows 10

#### Versions of xlwings, Excel and Python (e.g. 0.11.8, Office 365, Python 3.7)
Xlwings: 0.27.2
Office: 2211
Python: 3.11

#### Describe your issue (incl. Traceback!)
First I made a new excel file with the corresponding python file with the quickstart command.
After I edited the PYTHONPATH in my xlwings.conf, Xlwings couldn't find my file (see error below).

```python
Traceback (most recent call last):

  File "<string>", line 1, in <module>

ModuleNotFoundError: No module named 'viewer'

```

#### How I fixed it
First I made mistake in the path and got the following error:

```python

  File "<string>", line 1

    import xlwings.utils;xlwings.utils.prepare_sys_path("P:\\Projects\\viewer\\viewer.xlsm;C:\\PATH_TO_EXCEL\\Excel\\XLSTART\\xlwings.xlam;;;;P:\\Projects\\viewer\"); import viewer;viewer.main()
                                    
SyntaxError: unterminated string literal (detected at line 1)


```

The error appears to occur because the path to be added is at index 5, but the path is only added from the 6th path (previous ones seem to be used for connection to sharepoint). See the code below from Xlwings/Utils:

```python
def prepare_sys_path(args_string):
   # some code

    if args[6:]:
        paths += args[6:]

    if paths:
        sys.path[0:0] = list(set(paths))
```

In short: when I switch "P:\path" with ";P:\path" everything works as it should.
Now I'm wondering, how can I be the only one. Since others don't suffer from this, did I make a mistake during the installation?
I don't know what other info is usefull to attach.

#### Include a minimal code sample to reproduce the issue (and attach a sample workbook if required!)

Just enter a path to the .py file in PYTHONPATH. Just like: "P:\Projects\viewer I used xlwings.conf for the settings. No code required.

**Comment by fzumstein** [comment]
Yeah I can't replicate the issue. Just to be clear, if you want to run the code on an unchanged quickstart project, you don't have to change the PYTHONPATH at all.

**Comment by SirStepheno** [comment]
Hmm strange, thanks for the answer. Gonna try to make a project on a other device and check if it works.
